### ISSUE TITLE
Execute return value

### ISSUE THREAD

**Issue by eistin** [lxc/pylxd#191]
Is it possible just to get a return value of the execute command in a container ? 
Like 0 when it's ok, and 1 for an error. I don't need to get the stdout and the stderr.
I didn't find the return value in the metadata of the api response.

**Comment by rockstar** [comment]
LXD doesn't provide the return value. Please file a bug with LXD.

**Comment by tych0** [comment]
I think it does, https://github.com/lxc/lxd/blob/master/doc/rest-api.md#10containersnameexec says once the command is done, the "return" member of the operation metadata should be the return code.

**Comment by eistin** [comment]
Yes, that's why i asked the question.
I searched few hours a way to get this return value, but I didn't find.
A way to get the return value of a command in a container is to write on the output at the end of your command with:
```
echo "RC=$?"
```
And you can get your return value with the 're' lib:
stdout, stderr = container.execute([(your command + "; echo "RC=$?")])
re.search('RC=(.+?)\n', stdout).group(1)

I hope that will help some of you.
(ps: it would be nice to have a return value of the command from the API)
